screenshots/README.md

```diff
@@ -1,20 +1,13 @@
-Screenshots
-===========
+# Screenshots
 
-These screenshots are generated by running the selenium tests, e.g. using
+These screenshots are generated by running the following commands:
 
-    npm run test:e2e
+``` bash
+npm run showcase&
+npm run screenshots
+```
 
-That will override screenshots, but not delete files that are no longer
-generated by the test; in that case, run `rm -f screenshots/**/*.png` before
-running this script.
+That will generate screenshots in `screenshots/custom/`.
```

Comment:
Maybe directly ignore this folder so that people do not accidentally commit their `custom` screenshots?

Reply:
Neither do I, but I have a slight preference for adding it to the ignore file, just because I'm sure at some point this directory will show up in one of my PR diffs accidentally ... 😉

Reply:
mh I have a slight preference to not add it to the ignore file, because really it's not something that should live there for long (unlike for instance node_modules, etc). Will leave it out for now!